docs/src/main/sphinx/object-storage/file-system-local.md

```diff
@@ -0,0 +1,45 @@
+# Local file system support
+
+Trino includes support to access a local file system with a catalog using the
+Delta Lake, Hive, Hudi, or Iceberg connectors. The local file system must be a
+local mount point for a shared file system available on all cluster nodes.
```

Comment:
Probably would be good to detail what sort of file systems can be used here .. and how does that work with concurrent access from multiple workers .. will all that just work fine?